packages/fx-core/resource/package.nls.json

```diff
@@ -293,7 +293,7 @@
   "core.createProjectQuestion.projectType.copilotPlugin.label": "Copilot Plugin",
   "core.createProjectQuestion.projectType.copilotPlugin.title": "Copilot Plugin",
   "core.createProjectQuestion.projectType.copilotPlugin.placeholder": "Select an option",
-  "core.createProjectQuestion.projectType.customCopilot.detail": "Build intelligent chatbot in Microsoft Teams easily using Teams AI Library",
+  "core.createProjectQuestion.projectType.customCopilot.detail": "Build intelligent chatbot as your own copilot in Microsoft Teams easily using Teams AI Library",
```

Comment:
Any specific reason to mention 'as your own copilot'?

Reply:
To be consistent with the title "Custom Copilot", i.e., let users know the bot is built as their own copilot in Teams.

Reply:
Got it.
To improve readability, remove the word 'easily'.

Reply:
Updated.